cl/lib/pacer.py

```diff
@@ -274,8 +275,8 @@ def make_address_lookup_key(address_info):
 
      - Sort the fields alphabetically
      - Strip anything that's not a character or number
-     - Remove/normalize a variety of words that add little meaning and are often
-       omitted.
+     - Remove/normalize a variety of words that add little meaning and
+       are often omitted.
```

Comment:
For a second I thought PEP8 was catching hanging orphans or whatever it's called when one word is hanging alone on a line. :)